code/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
smart idea to have this in here!
testing coming soon, tm :)

Reply:
yesss